src/families/ripple/ScreenEditFee.js

```diff
@@ -27,7 +28,11 @@ type RouteParams = {
 };
 
 function RippleEditFee({ route }: Props) {
-  const { account } = useSelector(accountScreenSelector(route));
+  const { account: accountLike } = useSelector(accountScreenSelector(route));
+
+  if (!accountLike) return null;
+
+  const account = getMainAccount(accountLike);
```

Comment:
it's supposed to be `getMainAccount(accountLike, parentAccount)`
now if there is no case where this can happen, i suggest to just do `invariant(account.type==="Account"...`